bentoml/server/gunicorn_server.py

```diff
@@ -45,13 +45,13 @@ class GunicornBentoAPIServer(BentoAPIServer):
         def metrics_view_func(self):
             from prometheus_client import (
                 CONTENT_TYPE_LATEST,
-                CollectorRegistry,
                 generate_latest,
                 multiprocess,
             )
 
-            registry = CollectorRegistry()
+            registry = self.app.wsgi_app.collector_registry
```

Comment:
This is assuming that our WSGI app only has `InstrumentMiddleware`. Are there a way for us to get the correct WSGI app if along the line we have more middleware for model server?

Reply:
Maybe in the future implements a DispatcherMiddleware by path if we choose to have more middleware.